src/interfaces/store-interfaces/state-interfaces/name-request-interface.ts

```diff
@@ -1,7 +1,11 @@
+import { EntityTypes } from '@/enums'
+
 // Name Request State interface
 export interface NameRequestIF {
+  legalType: EntityTypes
+  legalName: string
   nrNumber: string
-  entityType: string
+  entityType: EntityTypes
```

Comment:
Why do we have both legalType and entityType?

If it's legit and the property can't be named something better, please add comments here.

Reply:
This i'm not sure of, name request  ( as it was for create ) utilized entityType i think? 
What the Correction schema looks for is `legalType`. I didn't look into removing entityType entirely or instances of where it is utilized as of yet.

Reply:
The schema specifies only `legalType`.

A sample IA I'm looking at uses `legalType`.

I think we should get rid of "entityType" from the NR and convert back and forth as needed (using parseFiling() and buildFiling()).

Reply:
Done and done 👍